stable/kommander/charts/kommander-karma/values.yaml

```diff
@@ -6,8 +6,10 @@
 # alertmanagerAddress: "HOST:PORT"
 alertmanagerAddress: ""
 # Kommander service account used to delete the karma configmap
+# Do not use, this will be deprecated
 kommanderServiceAccount: kommander-kubeaddons
```

Comment:
Should this not just get removed and the SA created above be used instead?

Reply:
yeah, I think it should be safe to just remove the value. I was following another old example but I think it's not necessary to add a deprecation warning here.